### PR TITLE
add profile screen and fix loading bug

### PR DESCRIPTION
**Pull request by koceljevac**

### CODE REVIEW

**Review by VojinBjelica — APPROVED** [review]

File: lib/core/utils/get_phone_number_localization.dart [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+
+String getPhoneNumber(String phoneNumber,BuildContext context) {
+  if (phoneNumber=="No number") {
+    final localization = AppLocalizations.of(context)!;
+    return localization.localeName == 'sr' ? 'Nema Broja' : 'No Number';
```
Comment (zeljkons1992) [inline lib/core/utils/get_phone_number_localization.dart:8]: Move this to localization constants

File: lib/features/profile/presentation/screens/profile_screen.dart [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            listener: (context, state) {
+              switch (state) {
+                case AuthLogoutSuccess _:
+                  return context.go("/login");
+              }
+            },
+            builder: (context, state) {
+              return BlocBuilder<ProfileBloc, ProfileState>(
+                builder: (context, state) {
+                  if (state is ProfileInitial) {
```
Comment (zeljkons1992) [inline lib/features/profile/presentation/screens/profile_screen.dart:41]: Always use switch for states

File: lib/languages/l10n/app_sr.arb [hunk 3]
```diff
@@ -23,5 +23,17 @@
     "vegaNoConfIde": "Nije potvrdjen Vega identitet",
     "googleStartVer": "Proces verifikacije je u toku",
     "googleVerSuc": "Proces verifikacije je uspesno zavrseno.\nBice te preusmereni na glavnu stranicu.",
-    "googleVerUns": "Proces verifikacije nije uspesno zavrsen.\nBice te preusmerenu na stranicu za prijavljivanje.\nOva aplikacija je samo za radnike Vega IT kompanije."
+    "googleVerUns": "Proces verifikacije nije uspesno zavrsen.\nBice te preusmerenu na stranicu za prijavljivanje.\nOva aplikacija je samo za radnike Vega IT kompanije.",
```
Comment (zeljkons1992) [inline lib/languages/l10n/app_sr.arb:26]: As we talked about, add shorter and more meaningful messages

File: lib/core/utils/get_phone_number_localization.dart [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+
+String getPhoneNumber(String phoneNumber,BuildContext context) {
+  if (phoneNumber=="No number") {
```
[the remark this answers is not in the document]
Comment (koceljevac) [reply lib/core/utils/get_phone_number_localization.dart:6]: UserProfileInformation userProfileInformation = UserProfileInformation(
        uid: user.uid,
        nameAndSurname: user.displayName!,
        email: user.email!,
        phoneNumber: user.phoneNumber ?? TextConst.noNumber,
        profileImage: user.photoURL!,
          dateTime: formattedDate,
          isEpsilon: true
      );

File: lib/languages/l10n/app_sr.arb (hunk 3 above)
Comment (koceljevac) [reply lib/languages/l10n/app_sr.arb:26]: yes sry

File: lib/core/utils/get_phone_number_localization.dart (hunk 4 above)
[the remark this answers is not in the document]
Comment (zeljkons1992) [reply lib/core/utils/get_phone_number_localization.dart:6]: Then it use TextConst.noNumber
`if (phoneNumber ==" TextConst.noNumber)`

**Review by zeljkons1992 — APPROVED** [review]

**Review by VojinBjelica — APPROVED** [review]

**Pull request merged**